app/components/allData.js

```diff
@@ -0,0 +1,70 @@
+'use client';
+import React from 'react';
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+import { useSelector } from 'react-redux';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+/**
+ * 
+ * @returns react component containing all of the weather data for each city entered.
+ */
+export const AllData = () => {
```

Comment:
This name is a bit to generic. Personally, prefer something more like specific that will allow me understand from the file name what the component is responsible for. i.e.
CityWeatherData